components/portfolio/PortfolioTechnologyTags.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PortfolioTechnologyTags = ({ tags, style }) => (
+  <div className="technology-tags-wrap">
+    <ul className="technology-tags--list">
+      {
+        tags.map((tag) => {
+          let href = '';
+          switch (tag.toLowerCase()) {
+            case 'meteor':
+              href = 'tech-back-end-meteor';
+              break;
+            case 'react':
+              href = 'tech-front-end-react';
+              break;
+            case 'react native':
+              href = 'tech-front-end-react-native';
+              break;
+            case 'mongo':
+              href = 'tech-data-base-mongo';
+              break;
+            case 'express':
+              href = 'tech-back-end-express';
+              break;
+            case 'redux':
+              href = 'tech-front-end-redux';
+              break;
+            case 'node js':
+              href = 'services-web-development-node';
+              break;
+            case 'cordova':
+              href = 'tech-apps-cordova';
+              break;
+            case 'angular':
+              href = 'tech-front-end-angular';
+              break;
+            case 'angular 2':
+              href = 'tech-front-end-angular-2';
+              break;
+            case 'android':
+              href = 'services-mobile-development-android';
+              break;
+            default:
+              break;
+          }
```

Comment:
I'd rather make an object technology-link and build logic on top of it.

Reply:
@epaminond Check now please.